localization/autoware_stop_filter/src/stop_filter_node.cpp

```diff
@@ -0,0 +1,84 @@
+// Copyright 2021 TierIV
```

Comment:
As this is newly added file, let's use `2025`
```suggestion
// Copyright 2025 TIER IV
```